lib/src/lib/dolos.ts

```diff
@@ -93,14 +107,16 @@ export class Dolos {
 
   public async analyzePaths(paths: string[], ignore?: string): Promise<Report> {
     let files = null;
+    let files_from_csv = null;
```

Comment:
Dolos uses mostly `camelCase` for variable names.

(I just now noticed that in the tests we are sometimes using `snake_case` by accident, apologies for the confusion :sweat_smile:)